Linux/Car_Client/chronoscomm.cpp

```diff
@@ -806,30 +809,50 @@ bool ChronosComm::decodeMsg(quint16 type, quint32 len, QByteArray payload, uint8
     } break;
 
     case ISO_MSG_OSEM: {
+        qDebug() << "OSEM";
```

Comment:
Might as well delete this unless it is needed

Reply:
True